src/Game.js

```diff
@@ -13,6 +16,25 @@ class Game {
   printQuestion(round) {
       util.main(round);
   }
+
+  start() {
+    let deck = this.buildDeck();
```

Comment:
Initially, I was instantiating a new instance of `Deck` by invoking `buildDeck` directly as an argument in the `beginRound`. However, for readability I set a new variable of `deck` equal to the deck that `buildDeck` returns and passed `deck` into `beginRound`. This makes the start function a lot easier to follow.